fdbserver/DataDistribution.actor.cpp

```diff
@@ -4860,7 +5199,20 @@ ACTOR Future<Void> storageRecruiter(DDTeamCollection* self,
 						    .detail("NumExistingSS", numExistingSS);
 					}
 
-					if (hasHealthyTeam && !tssState->active && tssToRecruit > 0) {
+					// check whether this candidate is a process under perpetual wiggling, but be paused because of
+					// unhealthy cluster status. For the data on this server is not completely moved to other team, it
+					// cannot be recruited as TSS. Otherwise, there's probability of data losing.
+					bool notForTSS = false;
```

Comment:
using a negative boolean is kind of weird (something like `!notForTSS` is kind of hard to read). Can you change this to something like `canBeUsedForTSS` or something else that is positive?